openfecwebapp/templates/partials/candidate/about-candidate.html

```diff
@@ -107,7 +107,7 @@ <h2 id="section-2-heading">About this candidate</h2>
       {% for committee in committee_groups['P'] | reverse %}
         <div class="callout callout--primary{% if loop.last %} u-no-margin{% endif %}">
           <h4 class="callout__title">
-            <a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
+            <a href="{{ url_for('committee_page', c_id=committee.committee_id) }}">{{ committee.name }}</a>
```

Comment:
Hmm.. in `constants.py`, is `DEFAULT_TIME_PERIOD` the most recent time period? If so then I can just omit passing cycle to url if it matches that in `committee_page`.